package-lock.json

```diff
@@ -54,7 +2356,8 @@
       "version": "5.3.2",
       "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-5.3.2.tgz",
       "integrity": "sha512-rq9s+JNhf0IChjtDXxllJ7g41oZk5SlXtp0LHwyA5cejwn7vKmKp4pPri6YEePv2PU65sAsegbXtIinmDFDXgQ==",
-      "dev": true
+      "dev": true,
+      "requires": {}
```

Comment:
po co ta zmiana?

Reply:
Nie mam pojecia skad to sie zmienilo. Jedyne co zrobilam to npm install na dwoch komputerach.